code/web/src/modules/user/api/actions.js

```diff
@@ -128,3 +130,22 @@ export function editDetails(newDetails) {
 		))
 	}
 }
+
+export function getDetails(user) {
```

Comment:
I wonder if this getDetails action creator could be combined with the editDetails action creator (similar to the structure of the login action creator above). I'm not sure if there would be any advantages to structuring it that way.